Shared/IdentifyController.h

```diff
@@ -73,12 +66,20 @@ private slots:
 private:
   bool addGeoElementPopup(Esri::ArcGISRuntime::GeoElement* geoElement, const QString& popupTitle);
 
+  bool m_isBusy = false;
   double m_tolerance = 5.0;
-  Esri::ArcGISRuntime::TaskWatcher m_layersWatcher;
-  Esri::ArcGISRuntime::TaskWatcher m_graphicsOverlaysWatcher;
   QList<Esri::ArcGISRuntime::PopupManager*> m_popupManagers;
 };
 
+// some shortcuts for working with multiple identify operation futures in a single 'QtFuture::whenAll' handler
+namespace IdentifyResultsVariant {
+  using FutureType = std::variant<QFuture<QList<Esri::ArcGISRuntime::IdentifyLayerResult*>>, QFuture<QList<Esri::ArcGISRuntime::IdentifyGraphicsOverlayResult*>>>;
+  namespace Types {
+    constexpr uchar LAYERS = 0;
+    constexpr uchar GRAPHICS = 1;
+  }
+}
+
```

Comment:
This seemed the most logical place since it is included also by the ContextMenuController tool. So far this is the only other tool using it besides the IdentifyController itself. I opted for this instead of creating something new.